frontend-web/src/app/api/auth/login/route.ts

```diff
@@ -0,0 +1,41 @@
+import { cookies } from 'next/headers';
+
+export async function POST(req: Request): Promise<Response> {
+  const { email, password } = await req.json();
+
+  try {
+    const response = await fetch('http://localhost:3000/api/auth/login', {
```

Comment:
mettre l'url dans une constante ou var d'env

```suggestion
    const response = await fetch('process.env.BACKEND_URL/api/auth/login', {
```